data/calculated_parser/functions.py

```diff
@@ -69,11 +70,293 @@ def sspeed(depth, latitude, temperature, salinity):
         press = [seawater.pres(d, latitude) for d in depth]
     except TypeError:
         press = seawater.pres(depth, latitude)
-
+    
     speed = seawater.svel(salinity, temperature, press)
     return np.array(speed)
 
 
+def sspeedmin(depth, lat, lon, temperature, salinity):
```

Comment:
Is the `lon` argument being used in this function?

Reply:
No